src/main/java/seedu/address/model/person/Tissuetype.java

```diff
@@ -0,0 +1,64 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's tissue type in ORGANice.
+ * Guarantees: immutable; is valid as declared in {@link #isValidTissuetype(String)}
+ */
+public class Tissuetype {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Tissue types are 6 integers from 1 to 12 separated by commas"
+                + " An example will be tt/1,2,3,4,5,6";
+    public final String value;
+
+    /**
+     * Constructs a {@code Tissuetype}.
+     *
+     * @param tissuetype A valid tissue type
+     */
+    public Tissuetype (String tissuetype) {
+        requireNonNull(tissuetype);
+        checkArgument(isValidTissuetype(tissuetype), MESSAGE_CONSTRAINTS);
+        value = tissuetype.toUpperCase();
```

Comment:
whats the use of uppercase since its number?

Reply:
Sorry will delete it